enclave_manager/avalon_enclave_manager/singleton/enclave_manager.py

```diff
@@ -347,13 +347,13 @@ def __update_receipt(self, kv_helper, wo_id, wo_json_resp):
 
 
 # -----------------------------------------------------------------
-def create_enclave_signup_data():
+def _create_enclave_signup_data():
```

Comment:
Is this function required ? Can't we directly call `enclave_info.EnclaveInfo.create_enclave_signup_data()`